PatternPal/PatternPal.Regonizers/Checks/MethodChecks.cs

```diff
@@ -64,7 +65,7 @@ public static bool IsInterfaceMethod(this IMethod methodSyntax, IEntity currentC
             return currentClass.ClassImplementsInterfaceMethod(methodSyntax);
         }
 
-        public static IEnumerable<string> GetCreatedTypes(this IMethod methodSyntax)
+        public static IEnumerable GetCreatedTypes(this IMethod methodSyntax)
```

Comment:
```suggestion
        public static IEnumerable<string> GetCreatedTypes(this IMethod methodSyntax)
```

Please add the generic argument back in.

Reply:
Done!